respa/urls.py

```diff
@@ -43,7 +44,7 @@
     url(r'^resource_image/(?P<pk>\d+)$', ResourceImageView.as_view(), name='resource-image-view'),
     url(r'^v1/', include(router.urls)),
     url(r'^v1/reservation/ical/(?P<ical_token>[-\w\d]+).ics$', ICalFeedView.as_view(), name='ical-feed'),
-    url(r'^$', RedirectView.as_view(url='v1/'))
+    url(r'^$', RedirectView.as_view(url='v1/')),
```

Comment:
This seems a bit unrelated change.. not a biggie, but better to separate unrelated changes/fixed into their own commits/PRs.